MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitSmm.c

```diff
@@ -22,8 +24,24 @@
 #include <Library/PerformanceLib.h>
 #include <Library/HobLib.h>
 
+EFI_STATUS
+EFIAPI
+SmmReadyToLockEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  );
+
+EFI_STATUS
+EFIAPI
+SmmReadyToBootEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  );
+
```

Comment:
Similar to PlatformInitStandaloneMm.c, it looks like there may be an opportunity to eliminate these prototypes.

Reply:
To eliminate these prototypes the functions would need to be removed which is possible.  The reason a wrapper function is used it due to convention for MM consolidation.

Reply:
I talked with Kun and will be removing these wrappers as they aren't necessary.  I don't know where I got the idea that they were a convention.

Reply:
Updated.